network-store-server/src/main/resources/iidm.cql

```diff
@@ -91,11 +91,13 @@ CREATE TABLE IF NOT EXISTS iidm.generator (
     position iidm.connectablePosition,
     minMaxReactiveLimits iidm.minMaxReactiveLimits,
     reactiveCapabilityCurve iidm.reactiveCapabilityCurve,
+    bus text,
```

Comment:
I known that is is going to be painful for updating CQL statements but this should be added just after node attribute...